tests/foreman/cli/test_roles.py

```diff
@@ -33,7 +23,7 @@ def test_positive_create_role_1(self):
         @BZ: 1138553
 
         """
-        for name in valid_names():
+        for name in generate_strings_list(length=10):
```

Comment:
I changed this to make this consistent with the UI tests.  Also now html tests will be included.